sh/test-data/helm/mongodb/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
This needs to have a licnece? Also if you can avoid checking it in the repo, it's preferred. You can curl it for testing,

Reply:
The bitnami charts use the Apache 2.0 license, but that's fair that we want to avoid checking it in. I now get it only for testing.